test_runner/src/main/kotlin/ftl/args/IosArgs.kt

```diff
@@ -49,6 +49,7 @@ IosArgs
       type: ${type?.ymlName}
       app: $app
       test-special-entitlements: $testSpecialEntitlements
+      fail-fast: $failFast
```

Comment:
It should be added also for Android

Reply:
@piotradamczyk5 @RainNapper include it in ICommonArgs .

Reply:
I believe this is already in common args? This code is for the `toString`, which references common args.